Makefile

```diff
@@ -167,6 +167,7 @@ coverage: .init
 test: .init
 	@echo Running tests:
 	@for i in $(addprefix $(SC_PKG)/,$(TEST_DIRS)); do \
+	  $(DOCKER_CMD) go test -i $$i && \
```

Comment:
Is there an easier way to run both commands at one time? This was the easiest thing that worked.